x/inflation/keeper/grpc_query_test.go

```diff
@@ -95,9 +95,7 @@ func (s *QueryServerSuite) TestQueryCirculatingSupply() {
 
 func (s *QueryServerSuite) TestQueryParams() {
 	nibiruApp, ctx := s.nibiruApp, s.ctx
-	resp, err := nibiruApp.InflationKeeper.Params(
-		sdk.WrapSDKContext(ctx), &inflationtypes.QueryParamsRequest{},
-	)
+	resp, err := nibiruApp.InflationKeeper.Params.Get(ctx)
```

Comment:
Since you have access to the full application, you could instantiate with NewQuerier to cover both the query server and internal keeper call with tests at the same time